eva-lib/src/main/java/uk/ac/ebi/eva/lib/models/ga4gh/GAKeyValue.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.lib.models.ga4gh;
+
+public class GAKeyValue {
```

Comment:
No idea, but probably this could be refactored. Still I would just leave it for a new ticket, otherwise this pr will be here until end of the world's day.